### PR TITLE
Add sleep related traits

### LINKED ISSUE #209: Map sleep-related EFO terms
- [x] add new terms
- [x] add mapped terms to [mapping table](https://docs.google.com/spreadsheets/d/13qh7dLE38vMyz91oRqj6GzKjFohNazNKAJxKG6Plw1o/edit#gid=1005741851)
Map 'sleep' related traits from [oba-efo-unreviewed.dosdp.tsv](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/mappings/oba-efo-unreviewed.dosdp.tsv):

subject_id | subject_label | match_type | quality | quality_label | attribute | attribute_label
-- | -- | -- | -- | -- | -- | --
EFO:0005273 | sleep depth | Lexical | PATO:0001595 | depth | GO:0030431 | sleep
EFO:0004870 | sleep measurement | Lexical | PATO:0000001 | quality | GO:0030431 | sleep
EFO:0005274 | sleep time | Lexical | PATO:0000165 | time | GO:0030431 | sleep
EFO:0007817 | sleep apnea measurement | Lexical | PATO:0000001 | quality | GO:0030431 | sleep
EFO:0008455 | sleep apnea measurement during REM sleep | Lexical | PATO:0000001 | quality | GO:0030431 | sleep
EFO:0004807 | short sleep | Lexical | PATO:0000574 | short | GO:0030431 | sleep
EFO:0008456 | sleep apnea measurement during non-REM sleep | Lexical | PATO:0000001 | quality | GO:0030431 | sleep
EFO:0005272 | sleep quality | Lexical | PATO:0000001 | quality | GO:0030431 | sleep
EFO:0005271 | sleep duration | Lexical | PATO:0001309 | duration | GO:0030431 | sleep
EFO:0005280 | sleep latency | Lexical | PATO:0001005 | latency | GO:0030431 | sleep

### PR DESCRIPTION
**Pull request by rays22**
If applied, this commit will fix #209.

### CODE REVIEW

File: src/patterns/data/default/entity_attribute.tsv [hunk 1]
```diff
@@ -4156,3 +4156,7 @@ OBA:2060170		UBERON:0002101	limb	PATO:0000140	position
 OBA:2060171		UBERON:0000966	retina	PATO:0000141	structure
 OBA:2060172		UBERON:0005056	external female genitalia	PATO:0000070	amount
 OBA:2060173		UBERON:0000970	eye	PATO:0000070	amount
+OBA:2040170		GO:0030431	sleep	PATO:0000001	quality
+OBA:2040171		GO:0030431	sleep	PATO:0001309	duration
+OBA:2040172		GO:0030431	sleep	PATO:0001005	latency
+OBA:2040173		GO:0030431	sleep	PATO:0002325	onset quality
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:4162]: How does PATO onset differ from HP onset?

https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/patterns/dosdp-patterns/disease_onset.yaml#L13

Comment (rays22) [reply src/patterns/data/default/entity_attribute.tsv:4162]: [HP:0003674 Onset](https://www.ebi.ac.uk/ols/ontologies/HP/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0003674) looks like it has a narrower and more human-specific meaning compared with [PATO:0002325 onset quality](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0002325).
The [PATO:0002325 onset quality](https://www.ebi.ac.uk/ols/ontologies/pato/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPATO_0002325)
seems to be a more generic term and I think more appropriate for traits. I think I would have used the PATO term even in the disease onset pattern.

Comment (rays22) [reply src/patterns/data/default/entity_attribute.tsv:4162]: I should note that [HP:0003674 Onset](https://www.ebi.ac.uk/ols/ontologies/HP/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0003674) which refers to an "age group in which disease manifestations appear" would not make sense in the 'sleep' process context.

Comment (matentzn) [reply src/patterns/data/default/entity_attribute.tsv:4162]: Thanks @rays22

**Review by ar-ibrahim — APPROVED** [review]

**Pull request merged**